src/components/ManagerPanelPage/ManagerPanelPage.tsx

```diff
@@ -0,0 +1,122 @@
+import AccountCircleIcon from '@mui/icons-material/AccountCircle';
+import MenuIcon from '@mui/icons-material/Menu';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import Toolbar from '@mui/material/Toolbar';
+import Tooltip from '@mui/material/Tooltip';
+import Typography from '@mui/material/Typography';
+import { MouseEvent, useState } from 'react';
+
+const pages = ['Orders', 'Menu', 'Tables', 'Reports'];
+const settings = ['Add new meal', 'Add new employee', 'Logout'];
+
+const useManagerPanelPage = () => {
+  const [anchorElNav, setAnchorElNav] = useState<null | HTMLElement>(null);
```

Comment:
Somehow, for me those `anchorElNav` and `anchorElUser` are not verbose enough. Maybe call them `anchorNavigationMenuEl`  and `anchorUserMenuEl` for readability?